app/src/main/java/co/runloop/influencer/data/ContactCursorWrapper.java

```diff
@@ -0,0 +1,45 @@
+package co.runloop.influencer.data;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.CursorWrapper;
+import android.provider.ContactsContract;
+import android.provider.ContactsContract.CommonDataKinds.Phone;
+import android.provider.ContactsContract.Contacts;
+
+import co.runloop.influencer.model.Contact;
+
+public class ContactCursorWrapper extends CursorWrapper {
+
+    private Context context;
```

Comment:
Or maybe even the phoneCursor instead? This could make the access modifier of the `PHONE_PROJECTION` array private, too

Reply:
I think this is not very good idea, we are requesting phoneCursor in process of foreach of first Cursor, if you want to put phoneCursor, it means make big query and keep it in memory, looks redurant

Reply:
Yes, that's a good point. 

Either way, the projection parameter should still be provided and not accessed directly, since this circular dependency suggests a bad design decision. Two classes should not know and use each other directly...

Reply:
Then, maybe should move projections to abstract contract class ?

Reply:
That could work too... I just don't entirely understand the design idea here...

It's defined in a separate class, but only used in this class. What is the benefit here? 
What if I'd like to use a different projection?

Reply:
I see now, that Phone projection can be moved to ContactsCursorWrapper class, it does not used in provider, it was, but now is not, and can be moved